makefile.shade

```diff
@@ -1138,11 +1138,18 @@ functions @{
         var sourceRuntime= Path.GetFileName(sourceDir);
         var destinationRuntime= sourceRuntime+ ".1.0.0-dev";
         var destinationDir = Path.Combine(FindRuntimePackagesFolder(), destinationRuntime);
-    
+
+        // Overwrite existing symbol link
         if (Directory.Exists(destinationDir)) 
         {
-            // Nothing to do, just return
-            return;
+            if (IsMono)
+            {
+                Exec("rm", "-rf " + destinationDir);
+            }
+            else
+            {
+                Exec("cmd", "/C rmdir " + destinationDir);
```

Comment:
1. Why don't you call `rmdir` directly?
2. Why not use `Directory.Delete` ?
3. Will `rm -rf` remove the contents of the symlink or just the symlink itself?